web/client/localConfig.json

```diff
@@ -37,6 +37,7 @@
             {
                 "name": "BackgroundSelector",
                 "cfg": {
+                    "bottom": 40,
```

Comment:
Yes, we could, but most probably this change in each installation by plugins presence and so on. 
So configure it only in the product is better choice, IMHO.